crnk-meta/src/test/java/io/crnk/meta/integration/AbstractMetaJerseyTest.java

```diff
@@ -66,6 +71,7 @@ public TestApplication() {
 			objectMapper.enable(SerializationFeature.INDENT_OUTPUT);
 			feature.addModule(createModule());
 			boot = feature.getBoot();
+			boot.setQuerySpecDeserializer(new DefaultQuerySpecDeserializer());
```

Comment:
that should not be neccessary => backward compatibility

Reply:
This is necessary since I can only set default impl in boot method, but here it's not called yet